WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackBrandingCoordinator.swift

```diff
@@ -6,7 +6,28 @@ class JetpackBrandingCoordinator {
     static func presentOverlay(from viewController: UIViewController, redirectAction: (() -> Void)? = nil) {
 
         let action = redirectAction ?? {
-            // TODO: Add here the default action to redirect to the jp app
+            #if DEBUG
+            let jetpackScheme = "jpdebug"
+            #elseif INTERNAL_BUILD
+            let jetpackScheme = "jpinternal"
+            #elseif ALPHA_BUILD
+            let jetpackScheme = "jpalpha"
+            #else
+            let jetpackScheme = "jetpack"
+            #endif
```

Comment:
We could probably move this into `AppConstants.swift`. It would require adding one for WordPress too though.

Reply:
Yeah, sounds good!